addons/account/models/account_journal.py

```diff
@@ -94,7 +94,7 @@ def _default_invoice_reference_model(self):
         comodel_name='account.account', check_company=True, copy=False, ondelete='restrict',
         string='Default Account',
         domain="[('deprecated', '=', False),"
-               "('account_type', '=', default_account_type), ('account_type', 'not in', ('asset_receivable', 'liability_payable'))]")
+               "'|', ('account_type', '=', default_account_type), ('account_type', 'not in', ('asset_receivable', 'liability_payable'))]")
```

Comment:
I don't know how ready this PR is, but is it something we really want? Letting a user put an asset type account as the default expense account seems dubious to me.

I suppose the reason you want that is to be able to have all the accounts for general type journal? I don't know if we want the receivables or the payables in it too.
If that's the case, I think we should define a domain explicitly for this case, like this for example
```suggestion
               "('account_type', 'not in', ('asset_receivable', 'liability_payable')), '|',
               ('account_type', '=', default_account_type), ('type', '=', 'general')]")
```

Reply:
Hello. Thanks for the feedback :)

This has been discussed with @william-andre when I noticed that the payroll configuration flow was broken due to the 2 commits pointed in the commit message.

I'm not an accounting expert, so I would like to have William's input about this :)

Have a nice day,

Yannick.

Reply:
So sproutch my remark? :smiling_face_with_tear: